spec/models/materialized_patient_summary_spec.rb

```diff
@@ -265,5 +265,37 @@ def refresh_view
 
       expect(described_class.find(not_ltfu_patient.id).ltfu?).to eq false
     end
+
+    it "is false if patient recorded a BP w/i the LTFU time" do
+      not_ltfu_patient = create(:patient, recorded_at: 2.years.ago)
+      create(:blood_pressure, patient: not_ltfu_patient)
```

Comment:
Let's be very deliberate with the dates we choose to record the BPs. If the cutoff is 365 days, we should have a BP recorded at 364 days and check that it's false. We should also have a BP recorded at 365 days and assert that it's true. (Same for the other associated records)

Reply:
Thanks, @harimohanraj89. It's an important point. Will tweak the dates accordingly.